keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/DockerArtifactSupplierTests.kt

```diff
@@ -96,10 +99,10 @@ internal class DockerArtifactSupplierTests : JUnit5Minutests {
         )
       }
 
-      test("supports Docker versioning strategy") {
-        expectThat(dockerArtifactSupplier.supportedVersioningStrategy)
+      test("supports Docker version sorting strategy") {
```

Comment:
Actually, this is not being used at this point, because the sorting strategy is automatically determined (as opposed to being configured in the delivery config and deserialized), which is where `supportedSortingStrategy` would come into play). I stopped short of removing all serialization support because I'm not sure if this will stay this way forever, but was inclined to do so. This is a question I'd like @robfletcher to weigh in on, though.

Reply:
oh I see... ok, thanks for explaining. Is there another place that you have tests for sorting by timestamp?

Reply:
Yes, here are the tests:
https://github.com/spinnaker/keel/blob/7acc44793f5a75866f14bbb5182af2ff435015f3/keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt#L353-L404

https://github.com/spinnaker/keel/blob/7acc44793f5a75866f14bbb5182af2ff435015f3/keel-core/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/CreatedAtSortingStrategyTests.kt

And here are the tests for the SQL changes to sort by timestamp and filter by branch/pull request, from #1564:
https://github.com/luispollo/keel/blob/2fb289cecbd034e57bcca3e412b3850867a1e746/keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt#L916-L1074